Jenkinsfile

```diff
@@ -40,9 +40,9 @@ pipeline {
                description: 'Comma separated list of boards')
         string(name: 'TESTS', defaultValue: 'all',
                description: 'Comma separated list of tests')
-        string(name: 'PRE_EXTRA_BUILD_ARGS', defaultValue: 'BUILD_IN_DOCKER=1',
+        string(name: 'PRE_EXTRA_BUILD_ARGS', defaultValue: 'DOCKER_MAKE_ARGS=-j BUILD_IN_DOCKER=1',
                description: 'Additional commands to use before make')
-        string(name: 'POST_EXTRA_BUILD_ARGS', defaultValue: '-j',
+        string(name: 'POST_EXTRA_BUILD_ARGS', defaultValue: '',
```

Comment:
Somebody might use `POST_EXTRA_BUILD_ARGS` later thinking that it will be passed to the build. This will only work when not using docker. Add a note somewhere?

Reply:
We don't have that warning in riot 😐